data/assets_vfsdata.go

```diff
@@ -0,0 +1,4717 @@
+		},
+		"/azurestack/cluster/master/outputs.tf": &vfsgen۰FileInfo{
+			name:    "outputs.tf",
+			modTime: time.Time{},
+			content: []byte("\x6f\x75\x74\x70\x75\x74\x20\x22\x69\x70\x5f\x61\x64\x64\x72\x65\x73\x73\x65\x73\x22\x20\x7b\x0a\x20\x20\x76\x61\x6c\x75\x65\x20\x3d\x20\x61\x7a\x75\x72\x65\x73\x74\x61\x63\x6b\x5f\x6e\x65\x74\x77\x6f\x72\x6b\x5f\x69\x6e\x74\x65\x72\x66\x61\x63\x65\x2e\x6d\x61\x73\x74\x65\x72\x2e\x2a\x2e\x70\x72\x69\x76\x61\x74\x65\x5f\x69\x70\x5f\x61\x64\x64\x72\x65\x73\x73\x0a\x7d\x0a"),
+		},
+		"/azurestack/cluster/master/variables.tf": &vfsgen۰CompressedFileInfo{
+			name:             "variables.tf",
+			modTime:          time.Time{},
+			uncompressedSize: 1487,
+
+			compressedContent: []byte("\x1f\x8b\x08\x00\x00\x00\x00\x00\x00\xff\x9c\x54\xbd\x8e\xdb\x30\x0c\xde\xfd\x14\x44\xa6\x76\xb9\xa9\xeb\x6d\x5d\x3
```

Comment:
The content of rhcos.json is the same as in the existing branch, but it should be different. 

The [instructions](https://github.com/openshift/installer-aro-wrapper/blob/main/docs/upstream-differences.md#update-installer-fork-again) need to be updated as they are not including the step to update the generated assets.

We'll be changing this next week.